ChequeWriter.DbInitializer/DbInitializer.cs

```diff
@@ -6,11 +6,8 @@ public static void Seed(string connectionString)
     {
         ArgumentNullException.ThrowIfNull(nameof(connectionString));
 
-        DbContextOptionsBuilder<ChequeWriterDbContext> optionsBuilder = new();
```

Comment:
Why this is removed ?
The reason : [EF Core](https://learn.microsoft.com/en-us/ef/core/cli/dbcontext-creation?tabs=vs)   
![image](https://github.com/kevinboon3288/cheque-writer/assets/38966644/b1c92f1f-da61-4e20-9447-8a629a7bf4f4)

The command in Package Manager Console can use as:
`Update-Database -Args '<connection string>'`